contrib/src/test/java/com/datatorrent/contrib/parser/JsonParserTest.java

```diff
@@ -0,0 +1,340 @@
+/**
```

Comment:
Just a thought..
If I understand correctly, this test won't be run as all the contrib tests are skipped by default.
Can parser tests be allowed to be executed as they do not require external setup?